store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreSchemaTest.java

```diff
@@ -70,95 +94,210 @@ public void setUp() throws Exception {
-                .merge(STRING_SCHEMA)
-                .build(), PROPERTIES);
+        // Element 1
+        fStore.execute(new AddElements.Builder()
+                .input(new Edge.Builder()
+                        .group("e1")
+                        .source("source1")
+                        .dest("dest1")
+                        .property("prop1", "value1")
+                        .build())
+                .build(), testContext);
 
+        // Element 2
+        fStore.execute(new AddElements.Builder()
+                .input(new Edge.Builder()
+                        .group("e1")
+                        .source("source1")
+                        .dest("dest2")
+                        .property("prop1", "value1")
+                        .property("prop2", "value2")
+                        .build())
+                .build(), testContext);
 
-        fStore.execute(new AddGraph.Builder()
-                .graphId("a")
+        // When
+        final CloseableIterable<? extends Element> a = fStore.execute(new GetElements.Builder()
+                .input(new EntitySeed("source1"))
+                .view(new View.Builder()
+                        .edge("e1", new ViewElementDefinition.Builder()
+                                .properties("prop2")
+                                .build())
+                        .build())
                 .build(), testContext);
 
-        fStore.execute(new AddGraph.Builder()
-                .graphId("b")
+        assertNotNull(a);
+        final Set<? extends Element> resultsSet = Streams.toStream(a).collect(Collectors.toSet());
+        final List<? extends Element> resultsList = Streams.toStream(a).collect(Collectors.toList());
+
+        // Then
+        HashSet<Edge> expected = new HashSet<>();
+        // Graph a, element 1
+        expected.add(new Edge.Builder()
+                .group("e1")
+                .source("source1")
+                .dest("dest1")
+                .matchedVertex(EdgeId.MatchedVertex.SOURCE)
+                .build());
+        // Graph a, element 2
+        expected.add(new Edge.Builder()
+                .group("e1")
+                .source("source1")
+                .dest("dest2")
+                .matchedVertex(EdgeId.MatchedVertex.SOURCE)
+                .build());
+        // Graph b, element 1
+        expected.add(new Edge.Builder()
+                .group("e1")
+                .source("source1")
+                .dest("dest1")
+                .matchedVertex(EdgeId.MatchedVertex.SOURCE)
+                .property("prop2", "")
+                .build());
+        // Graph b, element 2
+        expected.add(new Edge.Builder()
+                .group("e1")
+                .source("source1")
+                .dest("dest2")
+                .matchedVertex(EdgeId.MatchedVertex.SOURCE)
+                .property("prop2", "value2")
+                .build());
+
+        assertEquals(resultsSet, expected);
```

Comment:
expected is the 1st argument, actual is the 2nd.

Reply:
Weird, my brain works the other way round but fair